core/src/env_platform/env_service.bal

```diff
@@ -30,9 +30,9 @@ listener http:Listener ep0 = new (9090, config = {
     }
 });
 
-@openapi:ServiceInfo {
-    contract: "resources/openapi_v3.yaml"
-}
+// @openapi:ServiceInfo {
```

Comment:
Any reason why this is commented out?

Reply:
Else it'll regenerate the files at each build I think. @yashodgayashan?

Reply:
Resolved the issue and remove the comment.